patterns/pattern-1/images/mysql-test-node/files/metrics.sql

```diff
@@ -0,0 +1,92 @@
+-- 
```

Comment:
What would be the reason to duplicate this SQL file without using it from the product distribution?